pkg/asset/installconfig/openstack/validation/machinepool_test.go

```diff
@@ -31,6 +35,28 @@ func validMachinePool() *openstack.MachinePool {
 	}
 }
 
+func validMachinePoolSmallVolume() *openstack.MachinePool {
```

Comment:
```suggestion
func invalidMachinePoolSmallVolume() *openstack.MachinePool {
```
This case is meant to cause an error, so it should be invalid